app/views/users/show.html.erb

```diff
@@ -1,6 +1,17 @@
-+<h1>Hello User!</h1>
- +<br />
- +<p>Username: <%= @users.name %></p>
- +<p>Email: <%= @users.email %></p>
- +<p>Member since: <%= (@users.created_at).to_formatted_s(:long_ordinal) %></p>
- +<p>Last login: <%= time_ago_in_words(@users.last_sign_in_at) %></p> 
+<h1>Hello User!</h1>
+ <br />
+ <p>Username: <%= @user.name %></p>
+ <p>Email: <%= @user.email %></p>
+ <p>Member since: <%= (@user.created_at).to_formatted_s(:long_ordinal) %></p>
+ <p>Last login: <%= time_ago_in_words(@user.last_sign_in_at) %></p> 
+ <h1>
+    To do list:
+ </h1>
+  <table class="table table-striped">
+     <tbody>
+       <br/>
+         <%= render  @item %>
```

Comment:
Is this referring to a collection of multiple items or a single item?  If multiple the convention is to pluralize is `@items`.  You'll also obviously need to update your controller accordingly.

Based on the feedback I gave in `items/_items` view,  I would recommend using the [render collections feature](http://guides.rubyonrails.org/layouts_and_rendering.html#rendering-collections) there